### DISCUSSION TITLE
Using Fastify as reverse proxy

### DISCUSSION THREAD

**Discussion by eskimoquinn** [fastify/fastify#4763] [Q&A]
Hello there, I would like to use fastify as a reverse proxy. My current reverse proxy handles multipart form data, so a requirement for my new reverse proxy would need to handle this. I am aware that fastify-reply-from and fastify-multipart plugins are not compatible. Does anyone have any advice on creating a reverse proxy with fastify that handles multipart form data?

**Comment by mcollina** [comment]
How would you like to handle multipart data?
